llama_stack/apis/batch_inference/batch_inference.py

```diff
@@ -7,49 +7,30 @@
 from typing import List, Optional, Protocol, runtime_checkable
 
 from llama_models.schema_utils import json_schema_type, webmethod
-from pydantic import BaseModel, Field
+from pydantic import BaseModel
 
 from llama_stack.apis.inference import (
-    CompletionMessage,
+    ChatCompletionResponse,
+    CompletionResponse,
     InterleavedContent,
     LogProbConfig,
     Message,
+    ResponseFormat,
     SamplingParams,
     ToolChoice,
     ToolDefinition,
     ToolPromptFormat,
 )
 
 
-@json_schema_type
```

Comment:
I found a bunch of older cruft re: batch completion and misplaced types, etc. I fixed those in this update as well.